audinterface/core/feature.py

```diff
@@ -198,6 +198,8 @@ def process_file(
         Raises:
             RuntimeError: if sampling rates do not match
             RuntimeError: if channel selection is invalid
+            RuntimeError: if multiple frames are returned,
+                but ``win_dur`` is not set
```

Comment:
You only add the docstring for https://github.com/audeering/audinterface/pull/19 here, but I cannot see the `ValueError` for the duplicated index added to any docstring.

Reply:
> You only add the docstring for #19 here

Ups, I must have accidentally stashed those changes, thought I had committed them in #19.

> I cannot see the ValueError for the duplicated index added to any docstring.

I added those now